src/redux/reducer.js

```diff
@@ -5,10 +5,22 @@ var detailsReducer = require('./conference-details.js').detailsReducer;
 var permissionsReducer = require('./permissions.js').permissionsReducer;
 var sessionReducer = require('./session.js').sessionReducer;
 var tokenReducer = require('./token.js').tokenReducer;
+var rowReducer = require('./splash-rows.js').rowReducer;
+var activityReducer = require('./activity.js').activityReducer;
+var newsReducer = require('./news.js').newsReducer;
+var dismissReducer = require('./handle-dismiss.js').dismissReducer;
+var countReducer = require('./project-count.js').countReducer;
+var cacheReducer = require('./homepage-cache.js').cacheReducer;
 
 var appReducer = combineReducers({
     session: sessionReducer,
     token: tokenReducer,
+    rows: rowReducer,
+    activity: activityReducer,
+    news: newsReducer,
+    projectCount: countReducer,
+    handleDismiss: dismissReducer,
+    homepageCache: cacheReducer,
```

Comment:
Another case for having the reducer be the thing exported — we could put all the reducers in one directory and use webpack to glob them all together and combine them at once.